backend/expense/expense-service-contract-2.0.0.yaml

```
@@ -0,0 +1,229 @@
+          type: string
+        headCode:
+          type: string
+        amount:
+          type: number
+          format: double
+        type:
+          type: string
+          enum: [DEBIT, CREDIT]
+        additionalDetails:
+          type: object
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)